plugins/woocommerce-admin/client/products/fills/product-form-fills.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+import { useSelect } from '@wordpress/data';
+import {
+	EXPERIMENTAL_PRODUCT_FORM_STORE_NAME,
+	WCDataSelector,
+} from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { Fields } from './product-form-field-fills';
+import { Sections } from './product-form-section-fills';
+
+const Form = () => {
+	const { formData } = useSelect( ( select: WCDataSelector ) => {
+		return {
+			formData: select(
+				EXPERIMENTAL_PRODUCT_FORM_STORE_NAME
+			).getProductForm(),
```

Comment:
At some point we want to hook the loading of this data into the loading state of the product form so additional fields don't show up delayed, or we could load this data through the preloaded settings so it's always immediately available?

Reply:
It could be simple enough to throw in another `hasFinishedResolution` from this selector into the product form loading state. I'm not sure what adding them to the "preloaded settings" entails, but I believe we're trying to cut down on preloaded data in general?